src/embed_tests/pysethome.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Reflection;
+using System.Collections.Generic;
+using NUnit.Framework;
+using Python.Runtime;
+
+
+namespace Python.EmbeddingTest
+{
+    public class PySetHomeSet
+    {
+        [SetUp]
+        public void SetUp()
+        {
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+            PythonEngine.Shutdown();
+        }
+
+        [Test]
+        public void TestSetHome()
+        {
+            string homePath = @"C:\Python27\";
+            PythonEngine.PythonHome = homePath;
+            PythonEngine.Initialize();
+            Assert.AreEqual(PythonEngine.PythonHome, homePath);
+        }
+    }
+}

```

Comment:
@leomeuk you can get python path using this c# code which requires python interpreter:

```csharp
dynamic pyos = Py.Import("os");
string homepath = pyos.path.dirname(pyos.path.dirname(pyos.__file__));
```

Note that `PythonEngine.PythonPath` is also "broken" like `PythonEngine.PythonHome`:

```
> #r "C:\Python\Python27\Lib\site-packages\Python.Runtime.dll"
> using Python.Runtime;
> PythonEngine.Initialize()
> PythonEngine.PythonPath
Hosting process exited with exit code -1073740940.
Loading context from 'CSharpInteractive.rsp'.
```